### DISCUSSION TITLE
Multiple Sources

### DISCUSSION THREAD

**Discussion by jonrrivera** [muxinc/next-video#186] [Q&A]
Can I add multiple sources. Like combining MP4 and HLS. Or does it only let you use one src like I see in the [docs](https://next-video.dev/docs)?

**Comment by luwes** [comment]
thanks for the question, no currently it's not possible to add in multiple sources. The workflow is built around proving a raw source video file and the processing like generating multiple renditions is left up to the provider.

Did you have a specific use case in mind? 
Would the multiple sources have the same content or is it like an art direction thing?
